### DISCUSSION TITLE
IKEA E1524 when bound to other device won't update status to z2m and ha

### DISCUSSION THREAD

**Discussion by HerrieMan72** [Koenkk/zigbee2mqtt#10169] [Devices]
Hi,
I've noticed something weird, don't know if it's me doing something wrong or if this is an issue, I searched the forum but couldn't find any similar post for this.
I have several IKEA E1524 remotes (all using the latest firmware) that I have bound to one or more lights, this works ok.
But when this remote is bound to one or more lights the actual button-presses are no longer visible in zigbee2mqtt and also not in my home assistant sensors.
When I unbind the remote from the lights, the button-presses are visible in z2m and ha again, but obviously the lights are no longer controlled using the remote.
So it seems like somehow the data of the buttons is either send to the lights, or to z2m, but not to both.
Can this be changed ? e.g. is there some setting that I can use to get the button-pressed both directly controlling my lights, but also still see the actions in z2m and ha ?

**Comment by malaakso** [comment]
Bind the remote to the coordinator in addition to the lights.

**Reply by HerrieMan72** [reply]
Thanks for your suggestion, that did help a bit, but not completely.
I was able to also bind the remote to the coordinator for Levelctrl and onoff, and now these button-presses are also visible to z2m and ha, that's great.
But I wanted to use the left/right buttons on this remote to control other things in ha, the onoff/up/down already directly control the lights via binding. But these left/right buttons are still not propagated to z2m and ha, no matter which binding I try to make between the remote and the coordinator.
Is there something else I can do ? these left/right clicks are available to z2m and ha as long as I do not bind the remote to the light.

**Reply by malaakso** [reply]
Left and right clicks are in the scenes cluster, bind that as well.

If you've recently updated the Ikea firmware, you may need to remove the remote from zigbee2mqtt, restart, and re-join for the scenes cluster to appear.

**Reply by HerrieMan72** [reply]
Hmm, I don't have a scenes cluster to bind, see attached screenshot, any idea how I can bind the scenes cluster ?

![image](https://user-images.githubusercontent.com/34721776/146143265-32273d19-4e95-4945-ae2d-434b7bbb7e95.png)

**Reply by malaakso** [reply]
Did you try to remove, restart and re-join? That should trigger discovery of the new cluster.

**Reply by HerrieMan72** [reply]
Yes, indeed that did the trick.
I didn't remove/restart/re-add the remote after I did the ota firmware update, now also the scenes cluster is visible for binding and I see the left/right button-presses appearing in z2m and ha.
Thanks a lot for your help.
